README.md

```diff
@@ -192,6 +211,18 @@ Note that calling this instead of `close` is "safe" in that it will conditionall
 
 In case of success, returns `1`. In case of errors, returns `nil, err`. In the case where the conneciton is conditionally closed as described above, returns `2` and the error string `connection must be closed`.
 
+## set_proxy_options
```

Comment:
Should this function be added to the ToC?

Reply:
Yep, good spot, please add to the ToC.

Reply:
Added.